README.md

```diff
@@ -1,180 +1,32 @@
-# OCDS Extension Template Repository
+# Amendment rationale classifications
 
-[OCDS extensions](https://standard.open-contracting.org/latest/en/extensions/) can be used to declare additional fields, objects and sections that will be included in an OCDS file, but which are not in the core schema, and to add or extend codelists. When any OCDS publication cites an extension in its' package metadata, it should be validated against the schema with that extension applied. 
+Adds a field to the `Amendment` object to classify the rationale for the amendment.
 
-This repository provides a template of an extension that people can base their own on.
+## Legal context
 
-## Getting started
+In the European Union, this extension's fields correspond to [eForms BT-140 (Change Reason Code) and BT-200 (Modification Reason Code)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest).
```

Comment:
Change the correspondences sentence to use the common form.

Reply:
I think this is now the common form :-)